src/Kliva/Views/RecordPage.xaml.cs

```diff
@@ -41,14 +46,42 @@ private void OnViewModelPropertyChanged(object sender, System.ComponentModel.Pro
         {
             if (e.PropertyName.Equals(nameof(ViewModel.CurrentLocation), StringComparison.OrdinalIgnoreCase))
             {
-                TrackingMap.ClearMap<MapIcon>();
+                //ToDo PiNi: only do this the first time?
+                //and add button to re-center map?
+                //==> To discuss with team
+                CenterMap();
+
```

Comment:
Would suggest to redrawn center each time, but can't we move out the zoom level so that it well only be set once?